go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/google/btree v1.0.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
+	github.com/pkg/errors v0.8.0
```

Comment:
Hi @dvonthenen,

If `go.mod` was changed, shouldn't the hash in `go.sum` be different? Or was the package already included transitively?

Reply:
package was already included transitively